src/svg/SvgGithub.js

```diff
@@ -8,8 +8,8 @@ const SvgGithub = () => (
     xmlns="http://www.w3.org/2000/svg"
   >
     <path
-      fill-rule="evenodd"
-      clip-rule="evenodd"
```

Comment:
did these just not work before?

Reply:
I think they did but there were warnings about updating it to work with jsx. A similar example is needing to do `className` instead of `class` in jsx